examples/landing.yaml

```diff
@@ -77,3 +77,21 @@ spec:
       description: Some Iam thing
       icon: IAMIcon
       isExternal: false
+  navigationSegments:
+    - sectionId: inventory-partial
```

Comment:
This is just a nit, but I am wondering if we should rename the `sectionId` to `segmentId`. We do call them segments.

Reply:
@Hyperkid123 good point; done!